apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js

```diff
@@ -236,7 +236,7 @@ $(document).ready(() => {
   function renderSchoolInfo() {
     if (schoolInfoMountPoint) {
       ReactDOM.render(
-        <div style={{padding: 10}}>
+        <div style={{padding: 22}}>
```

Comment:
This div has its own in-line styling and `SchoolDataInputs` adds additional padding on top of that, so I combined them here and removed it from the stylesheet to make the school association look right on the new sign-up page (without this change, the school association had funky-looking spacing).